app/pir_alarm.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+from os import sys, path
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+
+from gpiozero import MotionSensor
+import time
+import subprocess
+
+from app import app
+from light_driver import LightDriver
+
+light_driver = LightDriver()
+
+pir = MotionSensor(app.config['PIR'])
+alarm_duration = app.config['ALARM_DURATION']
+radio_duration = app.config['RADIO_DURATION']
+alarm_file = app.config['ALARM_FILE_NAME']
+alarm_duration_seconds = (alarm_duration * 60)
+radio_duration_seconds = (radio_duration * 60)
+
+print("")
+print("Alarm Duration %s" %alarm_duration_seconds)
+print("PIR Pin %s" %pir)
+print("")
+
+"""
+Play Obnoxious Alarm Sound
+"""
+def main():
+    print("Started Motion Detection")
+    light_driver.on()
+    cmd = ['mpc', 'stop']
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)
```

Comment:
This is another file that could benefit from having a `run_process` function; I wonder if we should make a utils class and have every file that needs to use import `run_process` from another file?